src/twisted/protocols/ftp.py

```diff
@@ -736,10 +739,20 @@ class FTP(basic.LineReceiver, policies.TimeoutMixin, object):
     passivePortRange = xrange(0, 1)
 
     listenFactory = reactor.listenTCP
+    _encoding = 'latin-1'
 
     def reply(self, key, *args):
         msg = RESPONSE[key] % args
-        self.sendLine(msg)
+        self.sendEncodedLine(msg)
+
+
+    def sendEncodedLine(self, line):
+        """
+        (Private) Encodes and sends a line
```

Comment:
The convention in Twisted is that if a function is private, it's name should be
prefixed by an underscore.  However, sendEncodedLine( is not needed.  Instead, delete this function and modify sendLine()